### PR TITLE
Add EnvExecutor in cloud

### PR DESCRIPTION
**Pull request by cbourgois**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix MANAGER-5497
| License          | BSD 3-Clause

## Description

* use bootstrapApplication to retrieve configuration
* update config to build AngularJS constants dynamically
* use `@ovh-ux/manager-preloader` and remove old preloader
* update build script to be region agnostic

### CODE REVIEW

File: packages/manager/apps/cloud/package.json [hunk 1]
```diff
@@ -11,10 +11,7 @@
   "license": "BSD-3-Clause",
   "author": "OVH SAS",
   "scripts": {
-    "build": "yarn build:eu && yarn build:ca && yarn build:us",
-    "build:ca": "webpack --env.production --env.region=\"CA\" && mv dist dist-CA",
-    "build:eu": "webpack --env.production && mv dist dist-EU",
-    "build:us": "webpack --env.production --env.region=\"US\" && mv dist dist-US",
+    "build": "webpack --env.production",
```
Comment (cbourgois) [inline packages/manager/apps/cloud/package.json:14]: Only one build ❤️

**Comment by ovh-cds** [comment]
CDS Report build#14286.0 ✘  
* 
  * Build ✘

**Comment by antleblanc** [comment]
Rebasing on `master` branch.

File: packages/manager/apps/cloud/webpack.config.js [hunk 2]
```diff
@@ -32,75 +28,47 @@ function readNgAppInjections(file) {
...
+      files: [
+        { from: path.resolve(__dirname, './client/assets'), to: 'assets' },
+        { from: foundNodeModulesFolder('angular-i18n'), to: 'angular-i18n' },
+        {
+          from: path.resolve(__dirname, './client/**/*.html'),
+          context: 'client',
+        },
+      ],
     },
-    REGION ? Object.assign(env, { region: REGION }) : env,
```
Comment (antleblanc) [inline packages/manager/apps/cloud/webpack.config.js:81]: We noticed that `env` is now missing from the shared configuration.
Here is the patch associated with: #3427

**Pull request merged**